content/applications/inventory_and_mrp/quality/quality_management/quality_alerts.rst

```diff
@@ -1,81 +1,88 @@
+- If any actions should be taken to prevent the issue from occurring again, enter them in the
+  :guilabel:`Preventive Actions` tab.
+- In the :guilabel:`Miscellaneous` tab, select the :guilabel:`Vendor` of the product, specify when
+  the alert was assigned to a quality team in the :guilabel:`Date Assigned` field, and enter an email
+  address that will be CC'd on emails about the quality alert, in the :guilabel:`Email cc` field.
+
+.. image:: quality_alerts/alert-form.png
    :align: center
-   :alt: An example of a completed quality alert form.
-
-Add quality alerts during the manufacturing process
-===================================================
+   :alt: A quality alert form that has been filled out.
 
-Odoo enables manufacturing employees to create quality alerts within a work order without accessing
-the *Quality* module. From the work order tablet view, click the :guilabel:` ☰ ` hamburger menu
-icon in the top left corner and select :guilabel:`Quality Alert`.
-
-.. image:: quality_alerts/work-order-tablet-view-menu-button.png
-   :align: center
-   :alt: Access the work order menu.
+Manage quality alerts
+=====================
 
-The quality alert form can then be filled out as detailed in the previous section. After saving the
-form, a new alert will appear on the :guilabel:`Quality Alerts` dashboard that can be found through
-the :menuselection:`Quality --> Quality Control` menu.
+To view all existing quality alerts, navigate to :menuselection:`Quality --> Quality Control -->
+Quality Alerts`. By default, alerts are displayed in a Kanban board view, which organizes them into
+different stages based on where they are in the review process.
 
-Manage existing quality alerts
-==============================
+To move an alert to a different stage, simply drag and drop it on the desired stage. Alternatively,
+select a quality alert to open it, and then click the desired stage above the top-right corner of
+the quality alert form.
 
-By default, quality alerts are organized in a kanban board view. The stages of the kanban board are
-fully configurable and alerts can be moved from one stage to the next by dragging and dropping or
-from within each alert. Additional options are available for viewing alerts, including graph,
-calendar, and pivot table views.
+To create a new alert within a specific stage, click the :guilabel:`+ (plus)` button to the right of
+the stage name. In the new alert card that appears below the stage title, enter the
+:guilabel:`Title` of the alert, and then click :guilabel:`Add`. To configure the rest of the alert,
+select the alert card to open its form.
 
-.. tip::
-   Filter alerts based on diverse criteria like date assigned or date closed. Alerts can also be
-   grouped by quality team, root cause, or other parameters found under the :guilabel:`Filters`
-   button menu.
+.. image:: quality_alerts/alert-kanban.png
+   :align: center
+   :alt: The Quality Alerts page, displaying alerts in a Kanban view.
```

Comment:
This is totally up to you, but I feel like this Quality Alerts kanban can be expanded on just a *little* more.

![image](https://github.com/odoo/documentation/assets/23611711/ed15f6f8-c5ea-4cfe-9c12-816c55a74ecb)


Are the "new", "confirmed" "Actions confirmed" stages available by default for all databases? How do users create a new stage?

There's also this new "Automations" option after clicking the gear icon of each stage.
![image](https://github.com/odoo/documentation/assets/23611711/2e32cba7-5206-4d09-a125-74797a74a03c)

The doc is great as is, and my suggestions are to help make this doc accessible to those who've never worked with kanban before. (Though im not sure how many of those there are)

Feel free to ignore! (: